CHANGELOG.md

```diff
@@ -29,14 +49,19 @@ All notable changes to this project will be documented in this file.
   * #### Removed
 * ### NI-DCPower
   * #### Added
+    * Channel repeated capability
   * #### Changed
   * #### Removed
 * ### NI-FGEN
   * #### Added
```

Comment:
Script Triggers, Marker Events are two NI-FGEN and NI-HSDIO repeated capabilities. Don't know of others.